docs/cookbook/setting_up_multi_language_siteaccesses.md

```diff
@@ -1,86 +1,87 @@
 # Setting up multi-language SiteAccesses and corresponding translations
 
-## Description
+Combining translated content with multiple SiteAccesses successfully can be challenging when you are new to eZ Platform.
 
-Combining translated content with multiple SiteAccesses successfully can be a little tricky to get your head around when you are new to eZ Platform.
+To achieve this for the most typical setups you need to follow three steps:
 
-To successfully achieve this for the most typical setups you will need to do all 3 following items:
+1. [Create your translations](#create-a-new-translation) in the database via the eZ Platform back office.
```

Comment:
I generally agree, but since we already have "back office" in small letters elsewhere, let's leave it like this for now, and then modify the terms throughout at once.